apps/core/templates/core/_maintenance_or_offline.html

```
@@ -13,14 +13,14 @@ <h1 class="text-center">🚧 🚧</h1>
-            <p class="text-center text-muted">We are sorry for any inconveniences</p>
+            <p class="text-center text-text-secondary">We are sorry for any inconveniences</p>
         {% elif called_by_offline_view %}
-            <p class="text-center text-muted">yamsa needs internet to work</p>
+            <p class="text-center text-text-secondary">yamsa needs internet to work</p>
         {% endif %}
     </div>
     {% if called_by_offline_view %}
-        <div class="fixed-bottom z-0 d-flex justify-content-center mb-5">
-            <button class="btn btn-primary"
+        <div class="fixed bottom-0 left-0 right-0 z-0 flex justify-center mb-12">
+            <button class="inline-flex items-center px-4 py-2 bg-blue-500 text-white font-semibold rounded-lg hover:bg-blue-600 active:scale-95 transition-all"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)